app/models/planning_application.rb

```diff
@@ -259,6 +259,10 @@ def closed_requests
     validation_requests.select { |req| req.state == "closed" }
   end
 
+  def parsed_application_type
+    application_type == "lawfulness_certificate" ? "Certificate of Lawfulness" : "Full"
+  end
```

Comment:
I'm a bit nervous about saying that if it's not "lawfulness_certificate", then it's definitely "Full"? What if in the future we add a new type and then it's that?

I think this should map "lawfulness_certificate" to "Certificate of Lawfulness", "full" to "Full", and then if it's neither, we either error or call `.humanize` on whatever it is?

Reply:
Sounds like a much better idea!

Reply:
I've updated and added a spec - note there is no spec for the case where it's something other than full or lawfulness as those are the only values we currently allow.